extensions/post_serializer_extension.rb

```diff
@@ -56,6 +57,14 @@ def include_qa_has_votes?
       @topic_view && object.is_qa_topic?
     end
 
+    def qa_can_vote
+      scope.current_user.present? && (object.user_id != scope.current_user.id)
```

Comment:
I feel like this attribute isn’t really necessary, on the client side we can just disable the button if the post is is equal to the current user’s id

Reply:
I made it this way to lay the groundwork for more granular voting disabling based on `qa undo vote action window`. Currently the `qa undo vote action window` is not being respected and can be bypassed so this attribute will be used in my next PR.

Reply:
We don't need this anymore as we are not replying on client side button enabling/disabling.